hack/postrelease/test_manifests_archives.py

```diff
@@ -0,0 +1,54 @@
+import requests
+import tarfile
+
+from versions import RELEASE_VERSION
+
+urls = {
+    "docs": "https://github.com/projectcalico/calico/releases/download/{v}/release-{v}.tgz".format(
```

Comment:
These are both the same URL - is there a reason we need both?

Reply:
oversight